gobblin-utility/src/main/java/org/apache/gobblin/exception/QuotaExceededException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gobblin.exception;
+
+import java.io.IOException;
+
+
+public class QuotaExceededException extends IOException {
+
+  public QuotaExceededException(String message) {
+    super(message);
```

Comment:
where does this message above `Quota exceeded for flowgroup group1 on executor jobExecutor...` originate from? Is the format of the exception defined in another rb? Does it make sense to have the constructor for this exception contain some of that `Quota exceeded for...` and parameterize the rest?

Reply:
The message is defined in the `UserQuotaManager`, I think that's an interesting idea but I usually don't see Exception classes in general forcing formatted messages (in case this exception is used outside the scope of GaaS), hence I just pass the message up the stack. I'm open to changing it to be formatted though! Wonder what others think @ZihanLi58

Reply:
I think we should just rely on the exception class instead of the message itself. It's kind of confusing in the test to verify the whole message though, you might want to change the assert condition to just make sure response contains QuotaExceededException.class.getSimpleName() ?